pkg/util/oom/oom_linux.go

```diff
@@ -101,15 +108,21 @@ func (oomAdjuster *OOMAdjuster) applyOOMScoreAdjContainer(cgroupName string, oom
 				return os.ErrNotExist
 			}
 			continueAdjusting = true
-			glog.Errorf("Error getting process list for cgroup %s: %+v", cgroupName, err)
+			glog.V(10).Infof("Error getting process list for cgroup %s: %+v", cgroupName, err)
```

Comment:
Yeah. Its because of the inherent race between starting a container and applying oom score asynchronously.